src/app/pages/my-virtrolio/my-virtrolio.component.ts

```diff
@@ -91,8 +91,8 @@ export class MyVirtrolioComponent implements OnInit {
   shareLink() {
     if (this.canShare()) {
       this.navigator.share({
-        title: this.displayName + '\'s virtrolio!',
-        text: 'Visit this link to sign their virtrolio and send them a custom message!',
+        title: this.displayName + 'made a virtrolio!',
```

Comment:
This implies that the virtrolio is new every single time. I'm not the biggest fan of that wording.
**(Be sure to test this code first, I don't know if the number gets auto-coerced to a string)**
```suggestion
        title: this.displayName + '\'s virtrolio for' + MsgIoService.currentYear + '!',
```

Reply:
Can't test this unless it's on the beta website (need https:// for `navigator.share()` to work). I'll run a code inspection later to check that the code is fine.